stable/artifactory/templates/artifactory-database-secrets.yaml

```diff
@@ -1,4 +1,4 @@
-{{- if and  (not .Values.database.secrets) (not .Values.postgresql.enabled) }}
+{{- if or (and  (not .Values.database.secrets) (not .Values.postgresql.enabled)) .Values.database.password }}
```

Comment:
@woodcockjosh I'm not sure this change is required , `.Values.database.password` implies external postgres